Documentation/Administration/Docker/Production/Ports/Index.rst

```diff
@@ -0,0 +1,61 @@
+:navigation-title: Ports
+
+..  include:: /Includes.rst.txt
+.. _container-hosting-ports:
+
+===================================================
+Ports when running TYPO3 on container-based hosting
+===================================================
+
+..  include:: /Administration/Docker/Production/_Experimental.rst.txt
+
+When deploying TYPO3 to a container-based hosting platform, make sure that the
+container's **internal port 80** is correctly exposed to the outside.
+
+Most platforms handle this by routing incoming traffic to a fixed external
+port (for example, port 443 for HTTPS) and forwarding it to port 80 inside the
+container. You do **not** need to modify the TYPO3 image or change its
+internal port.
+
+Make sure your hosting configuration (such as ingress or service mapping)
+forwards traffic to port `80` inside the container.
+
+..  tip::
+
+    Most TYPO3 Docker images are configured to listen on port ``80`` inside the
```

Comment:
The webserver within the image is listening on port 80 by default. Not the image

Reply:
I rephrased it some. But you can kind of treat containers like black boxes. True there is a webserver within but that webserver becomes part of the image